root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
+    <!-- ko with: target() && target().label() -->
+      <!-- ko if: $data.gid -->
+        <p data-bind="html: $data.selectionMessage()"></p>
+      <!-- /ko -->
+      <!-- ko ifnot: $data.gid -->
+        <p>
+          [% l('The {doc|label} is the imprint associated with the release.
+                Please also see the guidelines about {style|special purpose labels}.',
+               { doc => { href => doc_link('Label'), target => '_blank' },
+                 style => { href => doc_link('Style/Unknown_and_untitled/Special_purpose_label'), target => '_blank' } }) %]
+        </p>
+      <!-- /ko -->
+    <!-- /ko -->
+  </div>
+
   <div class="bubble" data-bind="bubble: $root.barcodeBubble">
     <p>
-      [% l('Please enter the barcode of the release you are entering, see <a href="{url}" target="_blank">Barcode</a> for more information.',
-           { url => doc_link('Barcode') }) %]
+      [% l('The {doc|barcode} is the numeric identifier (such as UPC or EAN) associated with the release.',
+           { doc => { href => doc_link('Barcode'), target => '_blank' } }) %]
     </p>
     <p>
-      [% l('If you do not know whether the release has a barcode or not, just leave this blank.') %]
+      [% l('If you don’t know whether the release has a barcode or not, please leave this blank.') %]
     </p>
     <!-- ko with: target() && target().barcode -->
-    <p data-bind="visible: message, text: message"></p>
-    <p data-bind="visible: existing, html: existing"></p>
-    <p class="field-error" data-bind="visible: error() || confirmed()">
-      <label>
-        <strong>
-          <input type="checkbox" data-bind="checked: confirmed" />
-          [% l("I confirm that this is the barcode as it appears on the release.") -%]
-        </strong>
-      </label>
-    </p>
+      <!-- ko if: message -->
+        <p data-bind="text: message"></p>
+      <!-- /ko -->
+      <!-- ko if: existing -->
+        <p data-bind="html: existing"></p>
+      <!-- /ko -->
+      <!-- ko if: error() || confirmed() -->
+        <p class="field-error">
+          <label>
+            <strong>
+              <input type="checkbox" data-bind="checked: confirmed" />
+              [% l("I confirm that this is the barcode as it appears on the release.") -%]
+            </strong>
+          </label>
+        </p>
+      <!-- /ko -->
     <!-- /ko -->
   </div>
 
+  <div class="bubble" data-bind="bubble: $root.packagingBubble">
+    <p>
+      [% l('The {doc|packaging} attribute describes the outermost packaging of the release.',
+           { doc => { href => doc_link('Release/Packaging'), target => '_blank' } }) %]
+    </p>
+    <p>
+      [% l('For digital media releases, this should usually be “None”.') %]
+    </p>
+  </div>
+
   <div class="bubble" data-bind="bubble: $root.annotationBubble">
     <p>
-      [% l('The annotation field functions like a miniature wiki.') %]
+      [% l('The {doc|annotation} field functions like a miniature wiki.',
```

Comment:
I know this is mostly just keeping the existing text. I do wonder whether we need to clarify that this can often be left blank? (I've seen beginners entering whatever crap here)

Reply:
How about just adding "It’s fine to leave it empty." after this sentence?

Reply:
I think we could go a bit further:

The {doc|annotation} field functions like a miniature wiki.
It can be used to store data that does not fit into the other fields. It is fine to leave it empty.